torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2964,6 +2981,29 @@ def sample_inputs_entr(op_info, device, dtype, requires_grad, **kwargs):
                                     low=low,
                                     requires_grad=requires_grad)))
 
+
+def sample_inputs_i0_i1(op_info, device, dtype, requires_grad, **kwargs):
+
+    samples = (SampleInput(make_tensor((S,), device, dtype,
+                                       requires_grad=requires_grad)),
+               SampleInput(make_tensor((), device, dtype,
+                                       requires_grad=requires_grad)))
+
+    if dtype.is_floating_point:
+        if op_info.op != torch.special.i0e:
```

Comment:
code nit: two if statements in a row (combine them)

Reply:
Updated. Thanks!